src/components/ha-selector/ha-selector-attribute.ts

```diff
@@ -23,15 +23,21 @@ export class HaSelectorAttribute extends SubscribeMixin(LitElement) {
   @property({ type: Boolean }) public required = true;
 
   @property({ attribute: false }) public context?: {
-    filter_entity?: string;
+    filter_entity?: string | string[];
   };
 
+  get _filterEntity(): string | undefined {
+    if (!this.context?.filter_entity) return undefined;
+    return Array.isArray(this.context.filter_entity)
+      ? this.context.filter_entity[0]
+      : this.context.filter_entity;
+  }
```

Comment:
I don't really like this... should we even allow multiple in the UI? It is confusing with things like this where we only look at the first entity...

Reply:
That's what we had before in automations trigger/condition : 
```ts
entity_id: entityId ? entityId[0] : undefined,
```

Reply:
UI supports multiple entities.

![CleanShot 2023-02-08 at 15 28 05](https://user-images.githubusercontent.com/5878303/217558372-a8e6004a-8680-4678-91a7-81916d4b15ca.png)

Reply:
I removed the usage of filter entity/attributes for multiple entity_id